src/graphql/fragments/postFieldsFragment.js

```diff
@@ -1,3 +1,24 @@
+// :TODO: clean this up and use proper query fragments?
```

Comment:
this seems fine to me, not quite sure the advantages of real fragments

Reply:
Yeah, it's a discussion but I know of no material advantages today in our stack for using real fragments vs JS wrapped ones in this way other than explicit graphql syntax validation. However, there is as of the last time I checked one big disadvantage to using native graphql (fragments or othewise) in our setup which is that `.graphql` files don't recompile within the dev server so whenever adding or updating them you need to restart the dev server (`yarn start`) to have the changes. That gotcha can ruin an afternoon if you're not aware of it.

Reply:
Yeah, I found that one already. All good!